tests/e2e/default/contacts/contact-details.wdio-spec.js

```diff
@@ -32,13 +33,27 @@ describe('Contact details page', () => {
     const parent = placeFactory.place().build({ _id: 'dist1', type: 'district_hospital' });
     const user = userFactory.build({ username: 'offlineuser', roles: [role] });
     const patient = personFactory.build({ parent: { _id: user.place._id, parent: { _id: parent._id } } });
-    const report = reportFactory
-      .report()
-      .build(
+
+    const reports = Array
+      .from({ length: 60 })
+      .map(() => reportFactory.report().build(
         { form: 'pregnancy_danger_sign' },
-        { patient, submitter: user.contact, fields: { t_danger_signs_referral_follow_up: 'yes' },
+        {
+          patient,
+          submitter: user.contact,
+          fields: { t_danger_signs_referral_follow_up: 'yes' },
         }
-      );
+      ));
+
+    const pregnancyReport = [
+      pregnancyFactory.build({
+        fields: {
+          patient_id: patient._id,
+          patient_uuid: patient._id,
+          patient_name: patient.name,
+        },
+      }),
+    ];
```

Comment:
```suggestion
    const pregnancyReport = pregnancyFactory.build({
        fields: {
          patient_id: patient._id,
          patient_uuid: patient._id,
          patient_name: patient.name,
        },
      });
```
Because it only needs one pregnancy report and the variable name is singular, it seems better to leave it like this because no more reports will be added to this variable.

Reply:
Thanks, now I understand Diana's  comment